chapter_convolutional-neural-networks/lenet.md

```diff
@@ -380,11 +380,21 @@ class TrainCallback(tf.keras.callbacks.Callback):  #@save
 
 #@save
 def train_ch6(net_fn, train_iter, test_iter, num_epochs, lr,
-              device=d2l.try_gpu()):
+              device=d2l.try_gpu(), mirrored=True):
```

Comment:
@mli I noticed that you removed this argument in https://github.com/d2l-ai/d2l-en/commit/3b2034c41efb226cae141796cdd1e23c7888ff2d. However, this should fail the batch norm scratch implementation as it's required in that notebook (see my comment [here](https://github.com/d2l-ai/d2l-en/pull/1054/files#diff-cf23455381e755a4d0f02d266b186b6eR533-R536)) and the CI somehow fails to catch that (perhaps d2l/tensorflow.py wasn't updated). I am adding this back for now.